config/Makefile.mingw

```diff
@@ -156,7 +156,7 @@ MKEXE_BOOT=$(CC) $(CFLAGS) $(LDFLAGS) $(OUTPUTEXE)$(1) $(2)
 MKEXE_ANSI=$(FLEXLINK) -exe
 
 ### How to build a static library
-MKLIB=rm -f $(1); $(TOOLPREF)ar rc $(1) $(2); $(RANLIB) $(1)
+MKLIB=rm -f $(1) && $(TOOLPREF)ar rcs $(1) $(2)
 #ml let mklib out files opts =
 #ml   Printf.sprintf "rm -f %s && %sar rcs %s %s %s"
```

Comment:
This obviously wasn't changed as part of this GPR, but it would be better if mingw/msvc were using the `del` command, rather than `rm` (or, better yet, of the deletion were done more expressly in the compiler with a `Sys.remove` call)

Reply:
Good point!  Actually, this `mklib` definition is used in only one place, namely `tools/ocamlmklib.ml` line 287, and the library file to be created is deleted before calling the `mklib` command.  So, we can simplify the corresponding `#ml` configuration lines to not call `rm` nor `del`.

The other place in OCaml where a static library is built is `utils/ccomp.ml`, function `create_archive`.  There,  the destination file is also deleted first, and moreover `mklib` is not used, rather the use of `ar`+`ranlib` or of `link /lib` is hardcoded.